src/inbox/mod_inbox_utils.erl

```diff
@@ -16,21 +16,38 @@
 
 %%%%%%%%%%%%%%%%%%%
 %% DB Operations shared by mod_inbox_one2one and mod_inbox_muclight
--export([reset_unread_count/3,
+-export([maybe_reset_unread_count/4,
+         reset_unread_count/3,
+         maybe_write_to_inbox/5,
          write_to_sender_inbox/4,
          write_to_receiver_inbox/4,
          clear_inbox/1,
          clear_inbox/2,
          get_reset_markers/1,
          if_chat_marker_get_id/2,
+         has_chat_marker/1,
          has_chat_marker/2,
          fill_from_attr/2,
          wrapper_id/0,
          get_option_write_aff_changes/1,
          get_option_remove_on_kicked/1,
          reset_marker_to_bin/1,
-         get_inbox_unread/2]).
-
+         get_inbox_unread/2,
+         all_chat_markers/0
```

Comment:
Do we need this export?

Reply:
Actually, nope. Removing this export makes a ton of tests fail.